docs/spec/generics.rst

```diff
@@ -1761,6 +1763,352 @@ overloads for each possible rank is, of course, a rather cumbersome
 solution. However, it's the best we can do without additional type
 manipulation mechanisms.)
 
+.. _`type_parameter_defaults`:
+
+Defaults for Type Parameters
+----------------------------
+
+(Originally specified in :pep:`696`.)
+
+Default values can be provided for a TypeVar, ParamSpec, or TypeVarTuple.
+
+Default Ordering and Subscription Rules
+'''''''''''''''''''''''''''''''''''''''
+
+The order for defaults should follow the standard function parameter
+rules, so a type parameter with no ``default`` cannot follow one with
+a ``default`` value. Doing so should ideally raise a ``TypeError`` in
+``typing._GenericAlias``/``types.GenericAlias``, and a type checker
```

Comment:
```suggestion
a ``default`` value. Doing so may raise a ``TypeError`` at runtime, and a type checker
```
It's not the typing spec's job to say what the runtime should do.

Reply:
Good catch. I caught (and removed) one such statement, but I missed this one.